test/basic-querying.test.js

```diff
@@ -57,8 +60,10 @@ describe('basic-querying', function() {
       User.destroyAll(done);
     });
 
+    var id = isCassandraConnector ? uuidV4() : 1;
```

Comment:
For example, we can add a `generateId(User, defaultValue)` method to `DataSource` so that a connector can provide its customization.